models/service_helpers.go

```diff
@@ -88,6 +88,21 @@ func ServicesForAgent(q *reform.Querier, agentID string) ([]*Service, error) {
 	return res, nil
 }
 
+// ServicesForNode returns all Services for Node with given ID provides insights.
+func ServicesForNode(q *reform.Querier, nodeID string) ([]*Service, error) {
+	tail := fmt.Sprintf("WHERE node_id = %s ORDER BY service_id", q.Placeholder(1)) //nolint:gosec
```

Comment:
btw, @DexterHD @BupycHuk: what do you think about switching to plain placeholders? Like:
```go
structs, err := q.SelectAllFrom(ServiceTable, "WHERE node_id = $1 ORDER BY service_id", nodeID)
```

We used `q.Placeholder(1)` because we knew we are going to switch away from MySQL to either SQLite or PostgreSQL, and they have different placeholder syntax. We switched, and very unlikely to switch again, so we can make things clearer.